src/main/java/org/graphqlcrudjava/DatabaseConnection.java

```diff
@@ -3,14 +3,29 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.imageio.IIOException;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 import java.sql.*;
 
 public class DatabaseConnection {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseConnection.class);
 
+    private static  final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
     //Any Data Source Connection
-        public static Connection connect(String jdbcUrl, String driver, String username, String password) throws SQLException {
+        public static Connection connect() throws SQLException, IOException {
+            System.out.println("Enter JDBC URL : ");
```

Comment:
I would preffer if you will use environment variables here and document them as part of the PR.